samples/matter/window_covering/boards/nrf54l15pdk_nrf54l15_cpuapp_release.conf

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Multirole is the only currently supported role by SoftDevice.
+CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=y
+
+# TODO: Workaround to be removed once DFU will be supported on nRF54L.
+CONFIG_CHIP_OTA_REQUESTOR=n
+CONFIG_CHIP_QSPI_NOR=n
+
+CONFIG_FPROTECT=n
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_FPU=n
+CONFIG_PM=n
+CONFIG_HWINFO_NRF=n
+
+CONFIG_NRF_802154_TEMPERATURE_UPDATE=n
+
+# TODO: Workaround to be removed once Zephyr's CONFIG_FPROTECT is supported on nRF54L
+CONFIG_CHIP_FACTORY_DATA_WRITE_PROTECT=n
+
+# TODO: Workaround for buttons on nRF54L15 PDK in revision 0.2.x. To be removed for a PDK in higher revisions.
+CONFIG_DK_LIBRARY_BUTTON_NO_ISR=y
```

Comment:
This might help: https://github.com/nrfconnect/sdk-nrf/pull/14826

Reply:
Nice, So we can remove this workaround once that PR is merged. :)